### PR TITLE
Fixes Android Deploy

### PR DESCRIPTION
**Pull request by redreceipt**
Blocked by #188

**IMPORTANT: If this is for testing code in our npm dependencies, you should be pointing your PR at the `canary` or `next` branch. `master` should never rely on unreleased code.**

### CODE REVIEW

File: .github/workflows/deploy-android.yml [hunk 1]
```diff
@@ -38,5 +38,8 @@ jobs:
       env:
         APP_DATA_URL: ${{ secrets.APP_DATA_URL }}
 
+    - name: Decrypt Secrets
+      run: (cd .. && yarn unpack-secrets ${{ secrets.ENCRYPTION_PASSWORD }})
```
Comment (christinamegura) [inline .github/workflows/deploy-android.yml:42]: Please update to `yarn secrets`

**Review by christinamegura — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-android.yml (hunk 1 above)
Comment (redreceipt) [inline .github/workflows/deploy-android.yml:42]:
```suggestion
      run: (cd .. && yarn secrets -d ${{ secrets.ENCRYPTION_PASSWORD }})
```

Comment (redreceipt) [reply .github/workflows/deploy-android.yml]: good now

**Comment by redreceipt** [comment]
@christinamegura this one is ready again

**Review by christinamegura — APPROVED** [review]

**Pull request merged**